src/components/Artist.jsx

```diff
@@ -0,0 +1,21 @@
+import "./styling/Artist.css"
+
+export const Artist = ({ artists }) => {
```

Comment:
This component name indicates that there is only one artist. But it takes care of all of them. maybe call it Artists next time.